user-storage-simple/src/main/java/org/keycloak/quickstart/readonly/PropertyFileUserStorageProvider.java

```diff
@@ -92,46 +91,49 @@ public SubjectCredentialManager credentialManager() {
     }
 
     @Override
-    public UserModel getUserById(String id, RealmModel realm) {
+    public UserModel getUserById(RealmModel realm, String id) {
         StorageId storageId = new StorageId(id);
         String username = storageId.getExternalId();
-        return getUserByUsername(username, realm);
+        return getUserByUsername(realm, username);
     }
 
     @Override
-    public UserModel getUserByEmail(String email, RealmModel realm) {
+    public UserModel getUserByEmail(RealmModel realm, String email) {
+        // TODO Auto-generated method stub
         return null;
     }
 
-
     // CredentialInputValidator methods
 
     @Override
     public boolean isConfiguredFor(RealmModel realm, UserModel user, String credentialType) {
         String password = properties.getProperty(user.getUsername());
-        return credentialType.equals(CredentialModel.PASSWORD) && password != null;
+        return credentialType.equals(PasswordCredentialModel.TYPE) && password != null;
     }
 
     @Override
     public boolean supportsCredentialType(String credentialType) {
-        return credentialType.equals(CredentialModel.PASSWORD);
+        return credentialType.equals(PasswordCredentialModel.TYPE);
     }
 
     @Override
     public boolean isValid(RealmModel realm, UserModel user, CredentialInput input) {
-        if (!supportsCredentialType(input.getType()) || !(input instanceof UserCredentialModel)) return false;
+        if (!supportsCredentialType(input.getType()) || !(input instanceof UserCredentialModel))
+            return false;
 
-        UserCredentialModel cred = (UserCredentialModel)input;
+        UserCredentialModel cred = (UserCredentialModel) input;
         String password = properties.getProperty(user.getUsername());
-        if (password == null) return false;
+        if (password == null)
+            return false;
         return password.equals(cred.getValue());
     }
 
     // CredentialInputUpdater methods
 
     @Override
     public boolean updateCredential(RealmModel realm, UserModel user, CredentialInput input) {
-        if (input.getType().equals(CredentialModel.PASSWORD)) throw new ReadOnlyException("user is read only for this update");
+        if (input.getType().equals(PasswordCredentialModel.TYPE))
+            throw new ReadOnlyException("user is read only for this update");
```

Comment:
please revert formatting changes (and keep `CredentialModel.PASSWORD` -> `PasswordCredentialModel.TYPE`)